resources/rbac_roles.go

```diff
@@ -0,0 +1,258 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/cq-provider-k8s/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func RbacRoles() *schema.Table {
+	return &schema.Table{
+		Name:         "k8s_rbac_roles",
+		Description:  "Role is a namespaced, logical grouping of PolicyRules that can be referenced as a unit by a RoleBinding.",
+		Resolver:     fetchRbacRoles,
+		Multiplex:    client.ContextMultiplex,
+		DeleteFilter: client.DeleteContextFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"uid"}},
+		Columns: []schema.Column{
+			client.CommonContextField,
+			{
+				Name:     "type_meta_kind",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("TypeMeta.Kind"),
+			},
+			{
+				Name:     "type_meta_api_version",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("TypeMeta.APIVersion"),
+			},
```

Comment:
lets be consistent with the type_meta / object meta prefix removal, i.e type_meta_kind -> kind